client/components/TestQueueRun/index.jsx

```diff
@@ -93,8 +93,12 @@ class TestQueueRun extends Component {
     }
 
     updateRunStatus(status) {
+        let statusMap = {
+            'In Review': 'draft',
+            Published: 'final'
```

Comment:
Please change "Published Back to "Final" and then it should be good to go.

Reply:
@rmeritz all set!